opscode-pushy-client.gemspec

```diff
@@ -15,10 +15,10 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = PushyClient::VERSION
 
-  gem.required_ruby_version = '>= 2.5'
+  gem.required_ruby_version = '>= 2.7'
```

Comment:
If we're not changing the code do we actually need to bump this other than removing the upper bound?

Reply:
doesn't look like it. I hit an error related to the ruby version so I had bumped it, but that may have been an order of operations issue.

That said, if Chef 16 requires Ruby 2.7 it's not like this can get away with less?

Reply:
16 requires 2.6

Reply:
You could make it Ruby 2.6+ / Chef 15+. That way we can pull it into DK 4 without having to pin there.